### ISSUE TITLE
Hide ping/fps bar with chat too

### ISSUE THREAD

**Issue by punisherhaxball** [haxball/haxball-issues#1904]
Currently if binded we can hide the chat but the ping/fps bar at the left remains. And if you are playing at closer camera zooms such as 2.25x for example, that bar blocks game vision.

As you can see it takes some place on the screen and for some reason I can't focus on game when it's here. 😅
![image](https://user-images.githubusercontent.com/114198188/235375420-ad8bb6f9-3209-4d44-b8a4-faa3ab7fd946.png)

**Comment by basro** [comment]
Yeah, I planned to do this soon. It is already the case if you use the Chat width "full" setting.
